addons/frag/XEH_postInit.sqf

```diff
@@ -1,32 +1,46 @@
 #include "script_component.hpp"
 
-if (isServer) then {
-    GVAR(lastFragTime) = -1;
-    [QGVAR(frag_eh), LINKFUNC(frago)] call CBA_fnc_addEventHandler;
-};
-
-["CBA_settingsInitialized", {
-    if (!GVAR(enabled)) exitWith {};
-
-    // Register fire event handler
-    ["ace_firedPlayer", LINKFUNC(fired)] call CBA_fnc_addEventHandler;
-    ["ace_firedNonPlayer", LINKFUNC(fired)] call CBA_fnc_addEventHandler;
-    ["ace_firedPlayerVehicle", LINKFUNC(fired)] call CBA_fnc_addEventHandler;
-    ["ace_firedNonPlayerVehicle", LINKFUNC(fired)] call CBA_fnc_addEventHandler;
-
-    addMissionEventHandler ["EachFrame", {call FUNC(masterPFH)}];
-}] call CBA_fnc_addEventHandler;
-
-// Cache for ammo type configs
-GVAR(cacheRoundsTypesToTrack) = createHashMap;
-
-
-// Debug stuff:
-
-#ifdef DRAW_FRAG_INFO
-[] call FUNC(dev_startTracing);
+[
+    "CBA_settingsInitialized",
+    {
```

Comment:
Gotcha, I thought they were ok like that since I it was never brought up in #9728, and the case between the coding guideline's discussion about control flow statement bracket placement and array entries is different. Should be fixed in https://github.com/acemod/ACE3/pull/10157/commits/70606d411cb8fc8fea93e08c9760846615cae216, but feel free to push any other ones you see.